### DISCUSSION TITLE
如何指定网站访问前缀

### DISCUSSION THREAD

**Discussion by flycj** [tryzealot/zealot#1737] [Q&A] [answered]
如果指定网站前缀为 /app，那么是否可以通过 web.wxf.com/app 访问项目下的所有页面
当前已通过docker 部署能够通过 http://xxx.29.104.xxx:8089/debug_files 访问成功页面，但是通过nginx 反向代理 配置了 web.wxf.com/app ，然后能够访问到首页，但是出现报错了，报错问题如下：
1. 打开 web.wxf.com/app/debug_files 后 样式丢失，发现样式文件路径是 http://web.wxf.com/assets/application-ed016ccc.css 不是 http://web.wxf.com/app/assets/application-ed016ccc.css
2. 点击菜单后 跳转的页面是 web.wxf.com/debug_files ，没有把 /app 带上

请问如何配置呢？ 是否能够支持呢？ 期待回复！
以下是实际截图情况
![image](https://github.com/user-attachments/assets/f3ece378-9155-4da6-816f-397f09587c42)

**Comment by icyleaf** [comment, marked as answer]
你可以试试通过配置 `RAILS_RELATIVE_URL_ROOT=/app` 环境变量重启 zealot，这是 Rails 的[特性](https://guides.rubyonrails.org/configuring.html#deploy-to-a-subdirectory-relative-url-root)。我没有测试过希望得到测试后的结果 :D

另外还需要更新反代服务比如 nginx、caddy 等配置。
